docs/sphinx/using/cpp.rst

```diff
@@ -20,15 +19,15 @@ Here we see that we can define a custom :code:`struct` that is templated on a :c
 Our kernel expression is free to use this template parameter in the allocation of a 
 compile-time-known register of qubits. Within the kernel, we are free to apply various quantum operations, 
 like a Hadamard on qubit 0 :code:`h(q[0])`. Controlled operations are **modifications** of single-qubit 
-operations, like the :code:`x<cudaq::ctrl>(q[0],q[1])` operation to affect a controlled-X. We 
+operations, like the :code:`x<cudaq::ctrl>(q[0],q[1])` operation which implements a controlled-X gate. We 
 can measure single qubits or entire registers. 
 
 In this example we are interested in sampling the final state produced by this CUDA Quantum kernel. 
 To do so, we leverage the generic :code:`cudaq::sample` function, which returns a data type 
 encoding the qubit measurement strings and the corresponding number of times that string 
 was observed (here the default number of shots is used, :code:`1000`).
 
-To compile and execute this code, we run the following 
+To compile and execute this code, we run the following: 
```

Comment:
"The following example illustrates how to compile and execute this code."

Avoids the hanging colon, removes the we.

Reply:
the last commit fixes this.